### PR TITLE
Added in the ability for Oak's equipped items to be saved

### PR DESCRIPTION
**Pull request by Blade666132313**

### CODE REVIEW

File: oak.js [hunk 1]
```diff
@@ -104,14 +104,21 @@ var activateOakItem = function(id){
 	if(player.oakItemSlots == 1){
 		deactivateAllOakItems();
 		oakItemList[id].active = 1;
+		player.oakItemsEquipped.push(oakItemList[id]);
```
Comment (Ishadijcks) [inline oak.js]: Shouldn't you empty the oakItemsEquipped before pushing another item?

Comment (Blade666132313) [reply oak.js]: The emptying has been handled in the deactivateAllOakItems function, as that's where the active flag of the oakItemList is also reset back to 0

Comment (Ishadijcks) [reply oak.js]: Ah I see

**Comment by Ishadijcks** [comment]
It works great. The only thing I'd like to see is that you save the id/name of the oak item instead of the entire object. To save space in the savefile.

If you change that I'll merge it!

**Pull request merged**